### ISSUE TITLE
Different between mimic's `class` and `style` fields?

### ISSUE THREAD

**Issue by mjurczyk** [peterchon/mimic#1]
```
After going through the same insanity over and over again, I started to think "Why not use CSS property name as my class name?" Sure it's little more work, but I'll always be able to control the style by just adding or removing a class name. Plus, I don't have to keep coming up with class names that makes sense.
```

What is exactly the difference between this (in mimic):

``` html
<p class="margin-top:20 padding:10 bg-color:black border-radius:5 color:white align:right">Can you guess how I would render?</p>
```

And native `style` field:

``` html
<p style="margin-top:20px; padding:10px; background-color:black; border-radius:5px; color:white; align:right">Can you guess how I would render?</p>
```

Does it provide any advantage, or did I completely miss the point :P ?

**Comment by hl** [comment]
``` html
<p class="margin-top:20"></p>
```

this doesn't have to mean 20px, it can be 20em, 20rem or anything else

**Comment by mjurczyk** [comment]
Hm, still makes me wonder if this is worth the trouble just for the universal unit, in exchange taking quite a massive `.css` file into the codebase. Well, I'll make sure to check it out ;) !

**Comment by hl** [comment]
You should never include the full CSS file in your project. Take the Sass files and strip out everything you don't need. The author made 3 grids for example and I highly doubt you will need all three of them. Then again this is not a framework but more a way of working.

Have you seen the size of the default Bootstrap files :smile:

**Comment by mjurczyk** [comment]
Yeah, I remember bootstrap's size and, even more important, its clumsiness,.That's why I am not saying it is pointless, just asking in which use-cases it would shine the most brightly (if in any) ;P

**Comment by dan-gamble** [comment]
I'm with @mjurczyk on this one, this for utility classes seems a step too far. You might as well just use inline styles for the sake of not having to write `px`, `em` or `rem`

**Comment by peterchon** [comment]
Well, I think what I'm doing is an extreme case scenario, but something that you can do. Mimic makes sense when you need to make quick css changes without having to go back to the main stylesheet.

**Comment by dblotsky** [comment]
Is it really that slow to switch files and change a value? Surely, it's not worth to reacquire the _massive_ maintainability cost that classes removed in the first place.

**Comment by peterchon** [comment]
You tell me - It's not for me to decide these for you, it's merely there if you want to use it.

**Comment by mjurczyk** [comment]
Hah, quite harsh responses, I must say, sorry if I started it :(
@peterchon since there was quite a lot of work involved already, if I were you I'd try now to find the exact use-case for the project and transform it a little, to be the best for that case. If it is to be a table management CSS - concentrate on that. `class` simplifier? Cool, maybe let the use create his configuration for this classes via some script interface? Many options there!

**Comment by peterchon** [comment]
@mjurczyk thanks for that - I agree, I might have too released it too quickly. I'm currently working on a v1.1 that will address a lot of the concerns and the exact use-case scenario.
